app/views/katello/api/v2/ostree_branches/index.json.rabl

```diff
@@ -2,6 +2,10 @@ object false
 
 extends "katello/api/v2/common/metadata"
 
-child @collection[:results] => :results do
+sorted = @collection[:results].sort_by { 
```

Comment:
We don't really do sorting or manipulation of the results in the view layer in general. The previous sorting issues we looked at were addressed in the controller. Do you see any issue moving it there? You mentioned about a 'bulky if statement' but we can probably organize the code a little better to eliminate that concern (thank you for identifying that though).